.dotnet.azure/sdk/openai/Azure.AI.OpenAI/tests/FineTuningTests.cs

```diff
@@ -213,19 +209,20 @@ public async Task CreateAndDeleteFineTuning()
             TrainingFile = uploadedFile.Id
         }.ToBinaryContent();
 
-        ClientResult result = await client.CreateJobAsync(requestContent);
-        FineTuningJob job = ValidateAndParse<FineTuningJob>(result);
+        CreateJobOperation operation = await client.CreateJobAsync(requestContent, false);
+        FineTuningJob job = ValidateAndParse<FineTuningJob>(await operation.GetJobAsync(null));
```

Comment:
Curious: do you need to call `GetJobAsync` to issue another service request, or would passing operation.GetRawResponse give you the service response you're looking for?

Reply:
It seems I definitely can do that. It isn't obvious to me what the GetRawResponse does since there are multiple responses it could mean from the LRO. Is it the response from the creation? Or the response from the latest status check? Or the response from some other API call? ... I don't like the name "GetRawResponse"